packages/web-app-files/src/helpers/resources.ts

```diff
@@ -279,41 +279,41 @@ function addSharedWithToShares(shares) {
       previousShare?.storage_id === share.storage_id &&
       previousShare?.file_source === share.file_source
     ) {
-      if (ShareTypes.containsAnyValue(ShareTypes.authenticated, [share.share_type])) {
+      if (ShareTypes.containsAnyValue(ShareTypes.authenticated, [parseInt(share.share_type)])) {
```

Comment:
Note: owncloud-sdk will return properties casted to string, therefore we need to parse a lot